e2e/testinfra/terraform/common/registries.tf

```diff
@@ -32,3 +32,13 @@ resource "google_artifact_registry_repository" "gar" {
   description   = "A private AR repository used for Config Sync e2e tests"
   format        = "DOCKER"
 }
+
+resource "google_secure_source_manager_instance" "configsync-test" {
```

Comment:
Sorry just one more nit, the name of the tf resource should be more precise (e.g. `ssm-instance`)

To be clear the `instance_id` below doesn't need to change, just the tf resource name